sdk/android/skin/build.gradle

```diff
@@ -114,7 +114,7 @@ repositories {
 }
 
 task copyLibs(type: Copy) {
-    from new File(projectDir, '../../../vendor/Ooyala/OoyalaSDK-Android/OoyalaSDK.jar')
```

Comment:
what happened to the `OoyalaSDK.jar`? Shouldn't OoyalaVRSDK depend on OoyalaSDK?

Reply:
I created a new JAR file from our SDK branches. After we will update dev branch in android SDK  we will return previous path.